plugins/talk-plugin-toxic-comments/server/apiKey.js

```diff
@@ -0,0 +1,6 @@
+const apiKey = process.env.TALK_PERSPECTIVE_API_KEY;
+if(!apiKey) {
```

Comment:
Check out https://github.com/coralproject/talk/blob/master/config.js#L168-L172 to do instead, check if you're in test mode, if you are, don't panic.

Reply:
Ah.. 👍